### PR TITLE
Add more space below the Rust badge in the README

### PR DESCRIPTION
**Pull request by camelid**
The extra space makes it easier to read the README.

### CODE REVIEW

**Comment by steveklabnik** [comment]
I am not sure how I feel about adding an increasing amount of literal HTML in our markdown. I'm not going to block this, but I'm not sure that I think it should be merged either.

**Comment by camelid** [comment]
I understand where you're coming from, I don't love all the HTML either. It just feels cramped to me right now, so that's why I wanted a bit more space:

image

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
+<p>
 <a href = "https://www.rust-lang.org/">
 <img width = "90%" height = "auto" src = "https://img.shields.io/badge/Rust-Programming%20Language-black?style=flat&logo=rust" alt = "The Rust Programming Language">
 </a>
+</p>
```
Comment (camelid) [inline README.md:6]: If you don't want to add more HTML, what about a horizontal line? Like this:

```suggestion
<a href = "https://www.rust-lang.org/">
<img width = "90%" height = "auto" src = "https://img.shields.io/badge/Rust-Programming%20Language-black?style=flat&logo=rust" alt = "The Rust Programming Language">
</a>

---

```

**Comment by camelid** [comment]
I changed it to add a horizontal line (in Markdown) instead and I rebased atop master. I can close this if you still would rather not have it, but I think it makes it more readable.

**Comment by camelid** [comment]
I would actually prefer just a plain-text title (like how it used to be), but I figured other people liked it better the way it looks now.

**Comment by Dylan-DPC** [comment]
Closing this as there doesn't seem to be motivation to take this further from the author and the reviewer.

**Pull request closed without merging**